test/test-project/test-global.test.js

```diff
@@ -10,30 +10,31 @@ const strThatShouldBeMissing = '<div class="text-sm text-gray-700 mb-2">Correlat
 const templateName = '@asyncapi/html-template';
 const tempOutputResults = '../temp/integrationTestResult';
 const fileToCheck = 'index.html';
+const templateNotfoundInGlobalPackage = require('../../lib/logMessages');
```

Comment:
don't you think it is better to call it the same as in the main code? I just have a feeling that later it introduces a kind of duplication in the naming `templateNotfoundInGlobalPackage.templateNotFound`

Reply:
Ya , I think I have complicated this a bit just gonna change it as `logMessages`.